dspace-spring-rest/src/test/java/org/dspace/app/rest/DiscoveryRestControllerIT.java

```diff
@@ -52,16 +52,16 @@ public void rootDiscoverTest() throws Exception {
         //When we call this endpoint
         getClient().perform(get("/api/discover"))
 
-                //The status has to be 200 OK
-                .andExpect(status().isOk())
-                //The type has to be 'discover'
-                .andExpect(jsonPath("$.type", is("discover")))
-                //There needs to be a link to the facets endpoint
-                .andExpect(jsonPath("$._links.facets.href", containsString("api/discover/facets")))
-                //There needs to be a link to the search endpoint
-                .andExpect(jsonPath("$._links.search.href", containsString("api/discover/search")))
-                //There needs to be a self link
-                .andExpect(jsonPath("$._links.self.href", containsString("api/discover")));
+                   //The status has to be 200 OK
```

Comment:
I see a lot of unnecessary formatting change here that makes difficult to identify what has been really changed and will change the history of the file making difficult to find in future what has been changed by who for what and when.
Can you restore the original format please?

Reply:
The changes are specific to the entity type facet which has been configured